app/assets/javascripts/angular/controllers/summoner_controller.js.erb

```diff
@@ -1,5 +1,44 @@
 app.controller('SummonerCtrl', ['$scope', function($scope){
-  console.log("summonerctrl here", $scope);
-  $scope.summoner.name = "Harunoame";
-  $scope.summoner.level = 30;
+  // Summoner Information
+  $scope.summonerInfo = {};
+  $scope.summonerInfo.name = "Harunoame";
+  $scope.summonerInfo.level = 30;
+
+  // Tabs stuff
+  $scope.tabs = {};
+  $scope.tabs.changed = function(tabHeading){
+    $scope.tabs.status[tabHeading] = true;
+    $scope.tabs.updateInfo[tabHeading]();
+  };
+  $scope.tabs.deselected = function(tabHeading){
+    $scope.tabs.status[tabHeading] = false;
+  };
+  $scope.tabs.status = {};
+  $scope.tabs.status.profile = false;
+  $scope.tabs.status.runes = false;
+  $scope.tabs.status.masteries = false;
+  $scope.tabs.status.matches = false;
+  $scope.tabs.status.champions = false;
+  $scope.tabs.status.league = false;  

```

Comment:
i can do that, but its much easier for me to read. And create new stuff as it goes on..